functest/common_test.go

```diff
@@ -1,3 +1,5 @@
+// +build functest
```

Comment:
should we have copyright on the top?

Reply:
no, coz +build tag comment must be first line